### PR TITLE
Added basic version of the experiments api with UTs

### PR DESCRIPTION
**Pull request by mik-dass**
fabric8-hdd-bug-id: fabric8-hdd/supervisor#3
fabric8-hdd-bug-id: fabric8-hdd/supervisor#8
Signed-off-by: mik-dass mrinald7@gmail.com

### CODE REVIEW

**Comment by mik-dass** [comment]
@miteshvp , @anmolbabu , @dev-gaur Please review

File: main.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
 //  OpenShift sample Node application
 var express = require('express'),
-    bodyParser = require("body-parser"),
-    app     = express();
-    
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
```
Comment (anmolbabu) [inline main.js:7]: It seems from below that express itself provides what body parser does(Basically the json body parsing)?
Verified?

**Review by anmolbabu — COMMENTED** [review]
Please remove linter specific changes like spacing, tabs etc... as Devang's PR is ding that already

File: main.js (hunk 1 above)
Comment (mik-dass) [reply main.js:7]: Yes using Express >= 4.16.0, body parser has been re-added under the methods express.json() and express.urlencoded().
http://expressjs.com/en/api.html

**Comment by anmolbabu** [comment]
@mitchharpur Please review

**Comment by devang-gaur** [comment]
@mik-dass  remove .gitignore file ( check my comment at #8 to learn why ) and commit the code again.

We can leave this for now by labelling it as low priority, as per our discussion in the cabal. @anmolbabu kindly do the honours if you have the access to administer.

**Review by anmolbabu — COMMENTED** [review]
Parking this for now ...

**Pull request closed without merging**